R/Mexico.R

```diff
@@ -39,7 +39,9 @@ Mexico <- R6::R6Class("Mexico",
           self$country
         )
       )
-      self$region_codes <- covidregionaldata::mexico_codes
+      region_data <- mexico_codes
```

Comment:
I thought we had discussed having a single region code table in wide format organised as: 

```
tibble(local, region, code)
```

Have we decided we don't want to do that and continue with the nest nibble approach? If that is the case

Reply:
@kathsherratt to clarify `code` here was assumed to be a variable with the actual code in it not a nibble containing another lay of data.